back-office/shop-parameters/order-settings/order-settings.md

```diff
@@ -0,0 +1,111 @@
+
+As the creation of an account is optional, the password field is also optional.
+
+**When disabled**, guest visitors are not allowed to place an order without creating an account.
+
+During the checkout in front-office, at the first step (personal information), a tab "Create an account" is displayed with the following form:
+
+![order create an account](https://github.com/PrestaShop/prestashop-specs/blob/marionf-patch-2/img/Order_create_account.png)
+
+As the creation of an account is not optional, the password field is required.
+
+### Disable reordering option
+
+This toggle is on No by default.
+
+When enabled, a "reorder" link is displayed in the order history page of the customer account and in the order detail:
+
+![reorder list](https://github.com/PrestaShop/prestashop-specs/blob/marionf-patch-2/img/Reorder.png)
+
+![reorder detail](https://github.com/PrestaShop/prestashop-specs/blob/marionf-patch-2/img/Reorder_detail.png)
+
+### Minimum purchase total required in order to validate the order
+
+By default this field is set to 0.00
+
+The merchant can enter the minimum value (taxes excluded) required in the cart for the customer to be able to make an order
+
+If the value is > 0.00 and not reached in the shopping cart front-office, then a message is displayed in the shopping cart: "A minimum shopping cart total of XX (tax excl.) is required to validate your order. Current cart total is XX (tax excl.)." and the "Proceed to checkout" button is disabled.
+
+![minimum order value](https://github.com/PrestaShop/prestashop-specs/blob/marionf-patch-2/img/minimum_order_value.png)
+
+**Please note this value is not taken into account for an order created in the back-office:** the merchant is allowed to create an order with an amount < to the value defined. 
+
+### Recalculate shipping costs after editing the order
+
+This toggle is on Yes by default.
+
+When enabled, the shipping costs are automatically updated when the merchant edit an order and when the price of the order exceeds the carrier's range.
+
+### Delayed shipping
+
+This toggle is on No by default.
+
+This option isn't working, see issue https://github.com/PrestaShop/PrestaShop/issues/9781
+
+### Terms of service
+
+This toggle is on Yes by default.
+
+When enabled, the terms and conditions are displayed in front-office during the checkout at the payment step.
+To validated the order the customer has to check the box. Otherwise, the "place order" button is disabled.
+
+When disabled, the terms and conditions are not displayed in front-office at the payment step.
+
+### Page for terms and conditions
+
+This drop-down is enabled if the terms of service is enabled, and is disabled (not clickable) if the terms of service is disabled.
+When enabled, the merchant can select the page corresponding to the terms and conditions.
+All the enabled CMS pages are listed in the drop-down.
+
+## Gift
```

Comment:
Same question: is it also mentioned in the invoice?

Reply:
Yes, the wrapping costs are displayed in the invoice